test/elm/list/test.coffee

```diff
@@ -289,9 +289,13 @@ describe 'Includes', ->
   it 'should execute to false when tuple sublist is not in list', ->
     @tuplesNotIncluded.exec(@ctx).should.be.false()
 
-  it 'should return null if either arg is null', ->
-    should(@nullIncluded.exec(@ctx)).be.null()
-    should(@nullIncludes.exec(@ctx)).be.null()
+  it.skip 'should return true if right arg is null', ->
+    # TODO: currently returns false
+    should(@nullIncluded.exec(@ctx)).be.true()
+
+  it.skip 'should return false if left arg is null', ->
+    # TODO: currently returns null
+    should(@nullIncludes.exec(@ctx)).be.false()
```

Comment:
To ensure we're on the same page... you're skipping these because the current code actually produces the *wrong* answers and we don't want tests that assert them as being correct when they actually aren't?  And fixing the logic and re-enabling the tests would go into a different PR?

Reply:
@cmoesel Correct. The new expected values match HL7_CQLANG_R1_STU2_2017JUL.  

List Includes:
![image](https://user-images.githubusercontent.com/14879344/43603204-4d9b23d6-9660-11e8-8feb-1d3b8ff670b4.png)
List IncludedIn:
![image](https://user-images.githubusercontent.com/14879344/43603217-55f29906-9660-11e8-87b4-8fc1e7f08727.png)

Even if we fixed them to match the spec, those tests are currently being translated directly to `contains` and `In`, which have different behavior than `includes` and `includedIn` after the most recent errata to the spec.

For now I think it's best to skip these tests with the expected behavior matching the spec, then it's a future task to figure out what to do about the discrepancies (if anything).

Reply:
OK.  Sounds good to me.